apps/web/app/api/me/route.ts

```diff
@@ -30,4 +41,10 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     lastUpdate,
     wasCold: isCold,
   });
+
+  // Set custom header
+  response.headers.set("x-is-cold", isCold.toString());
```

Comment:
isCold should be `true` here, so the value would be "true"

Reply:
Discussed in DMS - already correct logic